homeassistant/components/meteo_france/__init__.py

```diff
@@ -152,6 +162,20 @@ async def _async_update_data_alert():
 
 async def async_unload_entry(hass: HomeAssistantType, entry: ConfigEntry):
     """Unload a config entry."""
+    _LOGGER.debug("Unload %s", entry.title)
+    if hass.data[DOMAIN][entry.entry_id][COORDINATOR_ALERT]:
+        hass.data[DOMAIN][
+            hass.data[DOMAIN][entry.entry_id][COORDINATOR_FORECAST].data.position.get(
+                "dept"
+            )
+        ] = False
+        _LOGGER.debug(
+            "Weather alert for depatment n° %s unloaded and released. It can be added now by another city.",
+            hass.data[DOMAIN][entry.entry_id][COORDINATOR_FORECAST].data.position.get(
+                "dept"
+            ),
```

Comment:
Save `hass.data[DOMAIN][entry.entry_id][COORDINATOR_FORECAST].data.position.get("dept")` as a var to avoid to long lines.

Reply:
done